hlx_statics/blocks/herosimple/herosimple.js

```diff
@@ -8,6 +8,9 @@ import {
  */
 export default async function decorate(block) {
 
+  const getBackgroundColor = block.getAttribute('data-background') || 'rgb(29, 125, 238)';
```

Comment:
```suggestion
  const backgroundColor = block.getAttribute('data-background') || 'rgb(29, 125, 238)';
```

because `getBackgroundColor` name suggests a function